src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,52 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int supplyTotalAmount = 0;
+        int buyTotalAmount = 0;
+        for (String line: readFromFile(fromFileName).split("\\r?\\n")) {
+            String[] row = line.split(",");
+            if (row[0].equals("supply")) {
+                supplyTotalAmount += Integer.valueOf(row[1]);
+            } else {
+                buyTotalAmount += Integer.valueOf(row[1]);
+            }
+        }
+        StringBuilder result = new StringBuilder();
+        result.append("supply," + supplyTotalAmount + System.lineSeparator());
+        result.append("buy," + buyTotalAmount + System.lineSeparator());
+        result.append("result," + (supplyTotalAmount - buyTotalAmount));
+        printToFile(toFileName, result.toString());
+    }
+
+    private String readFromFile(String fromFileName) {
+        File file = new File(fromFileName);
+        StringBuilder result = new StringBuilder();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(file))) {
+            int value = bufferedReader.read();
+            while (value != -1) {
+                result.append((char) value);
+                value = bufferedReader.read();
+            }
```

Comment:
what if we read not per symbol, but per line?
and after each line add System.fileSeparator?